kedro_azureml/cli_functions.py

```diff
@@ -147,4 +147,4 @@ def parse_extra_env_params(extra_env):
         if not re.match("[A-Za-z0-9_]+=.*", entry):
             raise Exception(f"Invalid env-var: {entry}, expected format: KEY=VALUE")
 
-    return {(e := entry.split("="))[0]: e[1] for entry in extra_env}
+    return {(e := entry.split("="))[0]: '='.join(e[1]) for entry in extra_env}
```

Comment:
nice catch, what do you think about
`return {(e := entry.split("=", maxsplit=1))[0]: e[1] for entry in extra_env}`?

Reply:
Great idea, will fix it